techniques/failures/F42.html

```diff
@@ -116,13 +116,13 @@ <h3>Scripting a <code class="language-html">div</code> element</h3>
          </ul>
       </section>
    </section><section id="related"><h2>Related Techniques</h2><ul>
-      <li><a href="../general/G115">G115</a></li>
       <li><a href="../failures/F59">F59</a></li>
+      <li><a href="../general/G115">G115</a></li>
    </ul></section><section id="resources"><h2>Resources</h2>
       
          <ul>
             <li>
-                  <a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA) Version 1.0</a>
+                  <a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA) Version</a>
```

Comment:
since we just want it to point at the most recent version, I have removed the word "Version"
```suggestion
                  <a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA)</a>
```